.github/workflows/release.yml

```diff
@@ -52,3 +55,16 @@ jobs:
           refreshToken: ${{ secrets.OAUTH_REFRESH_TOKEN }}
           force: true # We have checked the `dart pub publish --dry-run` `in build.yaml`, it's ok to force publish here.
           skipTests: true
+      - name: Build DartDoc 📖
+        run: |
+          # Generate DartDoc
+          dart doc
```

Comment:
I'm not very sure if the action `k-paxian/dart-package-publisher@master` brings the dart env or not. Maybe we need to test it first.

Reply:
I've aded a step to build.yml to attach it to the workflow. If that works, then this one should work the same…

Reply:
I think it's not the same, in the build.yaml, you have set up the dart env by using the action `uses: dart-lang/setup-dart@v1`, I think you should also add it here.